system/mp/mpcomm.cpp

```diff
@@ -2178,194 +2265,272 @@ void CMPConnectThread::startPort(unsigned short port)
     if (!listen)
         return;
     running = true;
+    if (acceptThreadPoolSize)
+    {
+        class CFactory : public CInterfaceOf<IThreadFactory>
```

Comment:
CFactory: The name could be more descriptive - what is it a factory for?

Reply:
Will rename to CMPConnectThreadFactory.
I do tend to use generic names if inline classes are used and therefore implicitly relevant to the surrounding code.